lib/utils/socketChainHelper.js

```diff
@@ -7,90 +7,91 @@ const {
 	normalizeErrorType,
 	responseMessageCode,
 } = require('./socketErrorMessages');
-const {config} = require('../../config');
-const logger = require('./logger');
 const t = require('../texts');
 
 const assertionErrorTypes = ['queryFailed', 'appRunning', 'appNotRunning', 'queryTimeout'];
 
 /**
  * Process network api websocket chain response
+ * @param logger
  * @param {ToStringFunc} toString
+ * @param {Boolean} continueOnFatalError
  * @returns {any}
  */
-const processServerResponse = toString => function processServerResponseHandler(res, data, jsonMessage) {
-	const isAssertionError = data.type === 'runSnippet' || assertionErrorTypes.includes(normalizeErrorType(res));
-	const responseForError = getResponseForError(res);
-	const message = getErrorMessage({
-		response: res,
-		toString,
-		jsonMessage,
-	});
-	const throwErr = err => {
-		err.stack = data.stack;
-		throw err;
-	};
-	const infoMessage = prefix => getInfoErrorMessage(message, prefix, res, data.stack);
-	const isSuccess = res.result === 'success' || res.result === 'warning';
-	const isEval = res.contentType === 'eval';
-	const isTestLine = res.contentType === 'testLine';
-	const isQuery = res.contentType === 'query';
-
-	// warnings
-	if (res.result === 'warning') {
-		logger.warn(infoMessage(''));
-	}
-	// fatal errors
-	if (res.result === 'fatal' && !config.continueOnFatalError) {
-		logger.error(infoMessage(''));
-		throwErr(new SuitestError(message), SuitestError.UNKNOWN_ERROR, responseForError);
-	}
-	// query
-	if (isQuery) {
+const processServerResponse = (logger, toString, continueOnFatalError) =>
```

Comment:
cant find place where processServerResponse accepts  continueOnFatalError

Reply:
cannot find it as well, @andriichumak @JeKachan is it possible that it was removed because of some BE issues?

Reply:
Let's drop `continueOnFatalError` option